frontend/projects/upgrade/src/app/shared/services/common-dialog.service.ts

```diff
@@ -4,7 +4,7 @@ import { MatConfirmDialogComponent } from '../components/mat-confirm-dialog/mat-
 import { DeleteFeatureFlagModalComponent } from '../../features/dashboard/feature-flags/modals/delete-feature-flag-modal/delete-feature-flag-modal.component';
 import { ImportFeatureFlagModalComponent } from '../../features/dashboard/feature-flags/modals/import-feature-flag-modal/import-feature-flag-modal.component';
 import { UpsertFeatureFlagModalComponent } from '../../features/dashboard/feature-flags/modals/upsert-feature-flag-modal/upsert-feature-flag-modal.component';
-import { UpsertPrivateSegmentListModalComponent } from '../../features/dashboard/segments/modals/upsert-private-segment-list-modal/upsert-private-segment-list-modal.component';
+import { UpsertPrivateSegmentListModalComponent } from '../../features/dashboard/segments-legacy/modals/upsert-private-segment-list-modal/upsert-private-segment-list-modal.component';
```

Comment:
Ah, can you move this component to the new `segments/modals` directory actually, and let vscode update the import paths? This was the new component developed during the FF stuff for adding / editing lists, so it belongs with the new stuff. I looked through the other components and I think this is the only one that needs to move.